modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -387,6 +387,31 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
     return { ...(await keychains.add(recipientKeychainParams)), reducedEncryptedPrv: reducedEncryptedPrv };
   }
 
+  /**
+   * Converts a User or Backup MPCv1 SigningMaterial to RetrofitData needed by MPCv2 DKG.
+   *
+   * @param decryptedKeyshare - MPCv1 decrypted signing material for user or backup as a json.stringify string and bitgo's Big Si.
+   * @param partyId - The party ID of the MPCv1 keyshare.
+   * @returns The retrofit data needed to start an MPCv2 DKG session.
+   */
+  getKeyDataForRetrofit(
```

Comment:
Nit: can you refactor the call sites of `getMpcV2RetrofitDataFromMpcV1Keys` to use this new fn instead?

Seems a little redundant to have both, we can use this as a more generic util function.

Reply:
yeah I'll line up a refactor pr